docs/content/latest/api/ysql/extensions.md

```diff
@@ -0,0 +1,361 @@
+---
+title: Extensions
+linkTitle: Extensions
+description: Install and Use Extensions
+summary: Reference for YSQL Extensions
+menu:
+  latest:
+    identifier: api-ysql-extensions
+    parent: api-ysql
+    weight: 4400
+isTocNested: true
+showAsideToc: true
+---
+
+This page documents how to install and use YSQL-compatible PostgreSQL extensions.
+
+
+## Use Included Extensions
+
+These are extensions that are included into the standard YugaByteDB distribution and can be enabled in YSQL by simply running `CREATE EXTENSION`.
+
+### pgcrypto
+
+The pgcrypto extension provides various cryptographic functions.
+
+<!-- 
+  Use explicit html for example throughout this file to exclude example 
+  subsections from the generated table of contents.
+-->
+<h4> Example </h4>
+
+```sql
+CREATE EXTENSION pgcrypto;
+CREATE TABLE pgcrypto_example(id uuid PRIMARY KEY DEFAULT gen_random_uuid(), content text, digest text);
+INSERT INTO pgcrypto_example (content, digest) values ('abc', digest('abc', 'sha1'));
+SELECT * FROM pgcrypto_example;
+```
+```
```

Comment:
Newline above.

See https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines.